test/test_cpp_extensions_open_device_registration.py

```diff
@@ -240,6 +240,12 @@ def test_open_device_storage():
             z2 = z2.foo()
             self.assertFalse(self.module.custom_add_called())
             self.assertTrue(z2.is_foo)
+            # check custom StorageImpl create
+            self.module.custom_storage_registry()
```

Comment:
This is not needed because this tests whether the registered storageImpl method is called correctly. The method constructed in the current test of storageImpl still adopts the existing basic building, so it essentially encapsulates a layer of calls

Reply:
There will not be any bad effect on subsequent UTs